processing/src/main/java/org/apache/druid/segment/StringDimensionIndexer.java

```diff
@@ -132,8 +132,10 @@ public long estimateEncodedKeyComponentSize(int[] key)
     // even though they are stored just once. It may overestimate the size by a bit, but we wanted to leave
     // more buffer to be safe
     long estimatedSize = key.length * Integer.BYTES;
-    for (int element : key) {
```

Comment:
nit: it's better to rename the original variable name `key` to `keys` to make it intuitive.

Reply:
fixed in d166364a1b76a487fc808d993973ed064cbf8978